src/sinol_make/helpers/compile.py

```diff
@@ -95,11 +96,19 @@ def compile_file(file_path: str, name: str, compilers: Compilers, weak_compilati
     os.makedirs(executable_dir, exist_ok=True)
     os.makedirs(compile_log_dir, exist_ok=True)
 
+    with open(os.path.join(os.getcwd(), "config.yml"), "r") as config_file:
+        config = yaml.load(config_file, Loader=yaml.FullLoader)
+
+    extra_compilation_files = [os.path.join(os.getcwd(), "prog", file)
+                               for file in config.get("extra_compilation_files", [])]
+    extra_compilation_args = config.get('extra_compilation_args', {}).get(os.path.splitext(name)[1], [])
```

Comment:
We can make this change also in other places in the code. What do you think?

Reply:
I agree that this is the better way. I changed it in other places, but couldn't find much